src/components/forms/registration/EmailField.tsx

```diff
@@ -23,6 +26,17 @@ export const EmailField: React.FC<Props> = (props) => {
           message: 'Bitte trage eine gültige E-Mail-Adresse ein!',
           validateTrigger: 'onSubmit',
         },
+        {
```

Comment:
We should perform those checks one after the other. Because an email address without an @ doesn't require to send this email to the server, since it is always a wrong email address. So using antd's `validateFirst={true}` on the `Form.Item` property should work here, I guess.

Reply:
Yeah, I wasn't sure how to do that. Thank you!